contents/node/nodejs.md

```diff
@@ -0,0 +1,209 @@
+# Introduction to Node.js
+
+Author: Rachael Sim
+
+# Overview
+*Node.js* is a "JavaScript runtime built on Chrome’s V8 JavaScript engine. Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient."
```

Comment:
seems like there's no JavaScript intro. would be good if someone can look into it too :D